### ISSUE TITLE
Could the Action Mouse Key be bound to force click with one finger of the trackpad of mac?

### ISSUE THREAD

**Issue by Prague2049** [rswgnu/hyperbole#385]
After using hyperbole, I think its function is very magical.
The trackpad of a Mac does not have a middle button. Could the Action Mouse Key of hyperbole be bound to force click with one finger of the trackpad of mac?

**Comment by rswgnu** [comment]
Yes, I believe so.  Look for a command named action-key or hkey-either and bind that to your Trackpad button push, assuming you have no need for drags.-- BobOn Sep 30, 2023, at 4:00 AM, Prague2049 ***@***.***> wrote:﻿
After using hyperbole, I think its function is very magical.
The trackpad of a Mac does not have a middle button. Could the Action Mouse Key of hyperbole be bound to force click with one finger of the trackpad of mac?

**Comment by Prague2049** [comment]
Thank you very much!This is very wonderful! 
`(define-key hyperbole-mode-map (kbd "<mouse-3>")  'action-key)`
I achieved this with the settings above.
I am very looking forward to when Hyperbole 9 will be released?

**Comment by rswgnu** [comment]
We hope soon, as it is close!

**Comment by Prague2049** [comment]
The very smart mouse drag function of Hyperbole left a very deep impression on me.
I have a suggestion about mouse dragging and link building, see if it can be implemented? 
I want to link the headline in one org file to the headline in another org file. When I drag the headline (include asterisks)of the source org file to the headline in the target org file  with the mouse, a link will be automatically established in the target org file and the org-id will be added to the headline of the source org file. 
The details as follows : 
The headline in the source org file is:
 *Item2
The headline in the target org file is:
 *Item1

After dragging,
The source org file is:
* Item2
:PROPERTIES:
:ID:       8E7514E0-0E2B-46D3-AB4A-F88F2720662E
:END:

The target org file is:
* Item1
[[id:8E7514E0-0E2B-46D3-AB4A-F88F2720662E][Item2]]

**Comment by rswgnu** [comment]
Interesting idea.  Let us think on it.

**Comment by Prague2049** [comment]
That would be cool!👍
